### PR TITLE
Added description of SearchEmbed and PinboardEmbed

### PR DESCRIPTION
**Pull request by bryanthowell-ts**
Short descriptions which specify the exact uses and differences between the two types of embed, aimed at users who are new to thoughtspot platform

### CODE REVIEW

File: docs/src/asciidocs/getting-started.adoc [hunk 1]
```diff
@@ -39,10 +39,20 @@ Based on integration requirements, import any of the following embed modules and
 
 * *SearchEmbed*
 +
-To xref:embed-search.adoc[embed the ThoughtSpot search] functionality.
+To xref:embed-search.adoc[embed the ThoughtSpot search] functionality. 
++
+The SearchEmbed component can be loaded up blank with a specified data source and no existing search, with a specified data source and a set of example search tokens, or with an existing saved Answer.
```
Comment (ShashiSubramanya) [inline docs/src/asciidocs/getting-started.adoc:44]: The **SearchEmbed** component can be loaded up blank with a specified data source and no existing search, with a specified data source and a set of example search tokens, or with an existing saved answer.

File: docs/src/asciidocs/getting-started.adoc [hunk 2]
```diff
@@ -39,10 +39,20 @@ Based on integration requirements, import any of the following embed modules and
 
 * *SearchEmbed*
 +
-To xref:embed-search.adoc[embed the ThoughtSpot search] functionality.
+To xref:embed-search.adoc[embed the ThoughtSpot search] functionality. 
++
+The SearchEmbed component can be loaded up blank with a specified data source and no existing search, with a specified data source and a set of example search tokens, or with an existing saved Answer. 
++
+In all cases, the search bar will appear along with full capabilites to edit and modify the search and the resulting chart or table.
```
Comment (ShashiSubramanya) [inline docs/src/asciidocs/getting-started.adoc:46]: capabilities

File: docs/src/asciidocs/getting-started.adoc [hunk 3]
```diff
@@ -39,10 +39,20 @@ Based on integration requirements, import any of the following embed modules and
...
+
 * *PinboardEmbed*
 +
-To embed ThoughtSpot xref:embed-a-viz.adoc[visualizations] and xref:embed-pinboard.adoc[pinboards].
+To embed ThoughtSpot xref:embed-a-viz.adoc[visualizations] and xref:embed-pinboard.adoc[pinboards]. 
++
+The PinboardEmbed component is used to embed both full pinboards with multiple answers, as well as a specific answer on a pinboard without the search bar (referred to as a "visualization").
```
Comment (ShashiSubramanya) [inline docs/src/asciidocs/getting-started.adoc:52]: You can use the **PinboardEmbed** component to embed a full pinboard with multiple answers, as well as a specific answer on a pinboard without the search bar. These answers are referred to as pinboard visualizations.

File: docs/src/asciidocs/getting-started.adoc [hunk 4]
```diff
@@ -39,10 +39,20 @@ Based on integration requirements, import any of the following embed modules and
...
+In all cases, the search bar will appear along with full capabilites to edit and modify the search and the resulting chart or table.
+
 * *PinboardEmbed*
 +
-To embed ThoughtSpot xref:embed-a-viz.adoc[visualizations] and xref:embed-pinboard.adoc[pinboards].
+To embed ThoughtSpot xref:embed-a-viz.adoc[visualizations] and xref:embed-pinboard.adoc[pinboards]. 
++
+The PinboardEmbed component is used to embed both full pinboards with multiple answers, as well as a specific answer on a pinboard without the search bar (referred to as a "visualization"). 
++
+There is currently no way to embed an individually saved answer outside of the SearchEmbed, so for a single visualization to be displayed, it must be on a pinboard and embedded uisng the PinboardEmbed with both pinboardId and vizId properties.
```
Comment (ShashiSubramanya) [inline docs/src/asciidocs/getting-started.adoc:54]: Suggesting an edit to break the paragraph into small sentences (to align with our writing standards). Please feel free to reword the sentence.

Although you can use a saved search answer as a data source in **SearchEmbed**,  ThoughtSpot does not support embedding a saved answer. However, you can embed a specific visualization from a pinboard using  `pinboardId` and `vizId` properties in **PinboardEmbed**.

**Review by ShashiSubramanya — CHANGES_REQUESTED** [review]
Added a few comments inline. Pls check.

File: docs/src/asciidocs/getting-started.adoc (hunk 1 above)
Comment (bryanthowell-ts) [reply docs/src/asciidocs/getting-started.adoc]: This has been changed and a new commit made to the branch

File: docs/src/asciidocs/getting-started.adoc (hunk 2 above)
Comment (bryanthowell-ts) [reply docs/src/asciidocs/getting-started.adoc]: This has been changed and a new commit made to the branch

**Review by bryanthowell-ts — COMMENTED** [review]
I think this resolves all of the issues raised

**Review by ShashiSubramanya — APPROVED** [review]
Looks good

**Pull request merged**